src/app/lorax/directives/modal-issue.tpl.html

```diff
@@ -0,0 +1,28 @@
+<article class="modal modal--fade-show modal-issue" ng-show="modalIssue.open" >
+  <a href="#" class="modal__close modal__close-absolute icon-close" data-lorax-prevent-default ng-click="modalIssue.closeModal()">Close</a>
```

Comment:
I agree, however, this is used as it is now in a few places so, I did not want to change that at the moment but, again, definitely something to rethink, and if for some reason that poses a significant issue, will at least add role=“button”